graf2d/gpad/v7/inc/ROOT/TVirtualCanvasPainter.hxx

```diff
@@ -44,9 +44,13 @@ protected:
 public:
   /// Default destructor.
   virtual ~TVirtualCanvasPainter();
-  
+
   virtual void AddDisplayItem(TDisplayItem *item) = 0;
 
+  virtual void AddMenuItem(const std::string &name, const std::string &title, const std::string &exec) = 0;
```

Comment:
It only makes sense when such TMenuItem will be available in Execute method.
For the moment it is not, therefore I would prefer to keep current interface for a while.

First of all, one should find solution how list of menu items can be preserved in the TDrawable 
and also correctly handled on the JS client side.